src/test/index.js

```diff
@@ -62,6 +62,7 @@ exports.auth = Object.freeze({
         return id;
     }),
     testWrite: createHeader(USERS.testWriteUser.username, USERS.testWriteUser.account_number, USERS.testWriteUser.tenant_org_id, false),
+    testWrite2: createHeader('testWriteUser2', 'testWrite2', USERS.testWriteUser.tenant_org_id, false),
```

Comment:
We should probably define this testWrite2 user in `src/connectors/users/mock.js` so it's easier to keep track of the overall test configuration.